### DISCUSSION TITLE
Pregunta archivos del proyecto

### DISCUSSION THREAD

**Discussion by davidlon13** [rramosp/ai4eng.v1#185] [UDEA Students]
Buenas tardes profe.

Nuestro compañero y yo tenemos una duda, lo que sucede es que para nuestro proyecto los archivos de datos que tenemos son archivos de Excel, y necesitamos una forma de subirlos a la nube para cumplir con los requisitos que usted nos pide para la revisión del proyecto. Intentamos hacer un repositorio de Github pero uno de los archivos pesa mas de 25M por o cual no nos permite subirlo.
Queríamos saber si nos puede dar alguna sugerencia para solucionar este problema y poder cumplir con los estándares para la revisión. 

Quedamos atentos y muchas gracias.

**Comment by Felipe-RA** [comment]
Saludos davidlon13

Si el archivo es demasiado pesado para trabajarlo, dividanlo en múltiples archivos más pequeños y presentenlo de forma ordenada
